lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -1541,6 +1541,13 @@ static Value createLinalgPayloadCalculationForElementwiseOp(
     Value result = convertScalarToDtype(b, loc, input, dtype);
     return result;
   }
+  if (isa<AtenAbsOp>(op))
+    return b.create<math::AbsOp>(loc, payloadArgs[0]);
+  if (isa<AtenReciprocalOp>(op)) {
+    Type elementType = payloadArgs[0].getType();
```

Comment:
Need to guard with float type check. Please refer to the example of relu above.

Reply:
Thanks for your comment!
done

Reply:
Btw, do we need a guard with float type check for Abs op?
I'm not very sure which kind of op need the guard. The "Tanh, Exp, Floor..." ops don't have guards...

Reply:
Yea, it depends on whether your code would be different which different type. int tanh and float tanh can share the same code.

Reply:
got it! thank you!